js_modules/dagster-ui/packages/ui-core/src/automation/MergedAutomationRoot.tsx

```diff
@@ -110,13 +109,38 @@ export const MergedAutomationRoot = () => {
     );
   }, [cachedData, visibleRepos]);
 
+  const tagsFilter = useDefinitionTagFilterWithManagedState({
+    allTags: useTagsForObjects(
+      repoBuckets,
+      useCallback((repoBucket) => {
+        return [
+          ...repoBucket.schedules.flatMap((schedule) => schedule.tags),
+          ...repoBucket.sensors.flatMap((sensor) => sensor.tags),
+        ];
```

Comment:
Is it ok for this to return 1000x copies of the same tag? It seems like that could happen here 👀

Reply:
`useTagsForObjects` dedupes them